### DISCUSSION TITLE
Send status (website widget)

### DISCUSSION THREAD

**Discussion by sbreiler-work** [chatwoot/chatwoot#2805] [Feature Requests]
Hey!

It doesn't seem to be possible to see, if a customer has read a sent message, is it? Can this be easily developed? :-)

**Comment by sbreiler-work** [comment]
Maybe the last ping (aka last seen) could be shown in the details?

**Comment by iammattmartin** [comment]
Very similar to an issue already raised. https://github.com/chatwoot/chatwoot/issues/2099

**Reply by sbreiler-work** [reply]
Thanks. I searched for similar issues but didn't found this one.

This discussion is then superfluous and can be closed.
